libs/server/Resp/ArrayCommands.cs

```diff
@@ -453,5 +453,65 @@ private bool NetworkArrayPING()
             WriteDirectLargeRespString(message);
             return true;
         }
+
+        private bool NetworkLCS<TGarnetApi>(ref TGarnetApi storageApi)
+            where TGarnetApi : IGarnetApi
+        {
+            if (parseState.Count < 2)
+                return AbortWithWrongNumberOfArguments(nameof(RespCommand.LCS));
+
+            var key1 = parseState.GetArgSliceByRef(0);
+            var key2 = parseState.GetArgSliceByRef(1);
+
+            // Parse options
+            var lenOnly = false;
+            var withIndices = false;
+            var minMatchLen = 0;
+            var withMatchLen = false;
+            var tokenIdx = 2;
+            while (tokenIdx < parseState.Count)
+            {
+                var option = parseState.GetArgSliceByRef(tokenIdx++).ReadOnlySpan;
+
+                if (option.EqualsUpperCaseSpanIgnoringCase(CmdStrings.LEN))
+                {
+                    lenOnly = true;
+                }
+                else if (option.EqualsUpperCaseSpanIgnoringCase(CmdStrings.IDX))
+                {
+                    withIndices = true;
+                }
+                else if (option.EqualsUpperCaseSpanIgnoringCase(CmdStrings.MINMATCHLEN))
+                {
+                    if (!parseState.TryGetInt(tokenIdx++, out var minLen) || minLen < 0)
```

Comment:
Doesn't look like there's an issue with setting minLen to a non-positive number, it just doesn't do anything

Reply:
It might be a miss from Redis side. Personally, I don't think we should inherit their flaws (Provided this is a flaw from Redis). Do you want to remove this check?

Reply:
I agree with you 100%, I'm just wondering if this might break any existing clients that rely on this logic for some reason...

Reply:
Done